lib/components/src/Zoom/Zoom.tsx

```diff
@@ -1,6 +1,6 @@
 import global from 'global';
-import { ZoomElement as Element } from './ZoomElement';
-import { ZoomIFrame as IFrame } from './ZoomIFrame';
+import { ZoomElement } from './ZoomElement';
+import { ZoomIFrame } from './ZoomIFrame';
```

Comment:
the dts-bundle-generator can't handle these types of renames well.. Until that issue is resolved, I'm just going to avoid them.. That's actually fairly easy.